doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -160,7 +160,10 @@ Thingy:53: Matter weather station
 Matter Bridge
 -------------
 
-|no_changes_yet_note|
+* Added:
+
+   The :kconfig:option:`CONFIG_BRIDGE_BT_MAX_SCANNED_DEVICES` kconfig option to set the maximum number of scanned Bluetooth LE devices.
+   The :kconfig:option:`CONFIG_BRIDGE_BT_SCAN_TIMEOUT_MS` kconfig option to set the scan timeout.
```

Comment:
Shouldn't these be bullets?

Reply:
Indeed, should be :)

Reply:
I will also update it on the main branch within the next possible PR.